templates/test/tr_job_result_details.html.ep

```diff
@@ -1,17 +1,18 @@
 %# 'bentostrap' has an item 'label' which
 %# we do not want to use so we use 'test-label'
-% my $bug = $res->{bug};
-% if ($bug) {
-    <span id="bug-<%= $jobid %>">
-        <a href="<%= bugurl_for $bug %>">
-            <i class="test-label <%= bugicon_for $bug %>" title="Bug(s) referenced: <%= $bug %>"></i>
-        </a>
-    </span>
-% }
-% elsif ($res->{label}) {
+% #if (my $bugs = $res->{bugs}) {
```

Comment:
This is a mistake, thanks for noticing. I removed the out-commented code in an additional commit since the PR is already merged.